collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
The original idea was if the team tempo plugin is not there then the database should not collect any team data. Instead the user will select the "ANY" field in the team dropdown. Right now with that code block a half baked attempt was made for backwards compatibility to store project data in both the project and the team repos. We figured we should remove that because it was confusing.

In reviewing and testing this code after being removed from it I realize now that that may be flawed as not having a team (ScopeOwnerCollectorItem) results in the UI not selecting a collector item. The "legacy" approach above may also be flawed as it may result in two dropdowns for "team" and "project" that are the same (both "project") and I don't remember if we had put in something for that case.

I will look more into this tomorrow.